WordPress/Classes/ViewRelated/Jetpack/Branding/Coordinator/JetpackFeaturesRemovalCoordinator.swift

```diff
@@ -61,4 +83,27 @@ class JetpackFeaturesRemovalCoordinator {
 
         return .normal
     }
+
+    /// Used to display feature-specific or feature-collection overlays.
+    /// - Parameters:
+    ///   - source: The source that triggers the display of the overlay.
+    ///   - viewController: View controller where the overlay should be presented in.
+    static func presentOverlay(from source: OverlaySource, in viewController: UIViewController) {
```

Comment:
I wonder if it would make more sense to call this `presentOverlayIfNeeded(from: in:)`. I'll leave the decision up to you though :)

Reply:
Agreed! Updated in 6426d1a9e2e273703cfebadfb152711795168a59